rosidl_generator_cpp/resource/msg__struct.hpp.template

```diff
@@ -144,6 +152,14 @@ for field in spec.fields:
   typedef std::weak_ptr<@(cpp_full_name)<ContainerAllocator> const>
     ConstWeakPtr;
 
+  // pointer types similar to ROS 1, use SharedPtr / ConstSharedPtr instead
+  typedef @(deprecated_macro_name)
+    std::shared_ptr<@(cpp_full_name)<ContainerAllocator>>
+    Ptr;
+  typedef @(deprecated_macro_name)
+    std::shared_ptr<@(cpp_full_name)<ContainerAllocator> const>
+    ConstPtr;
```

Comment:
I think leaving it where it is will be fine. I'd prefer the other location if not for Windows.